seedvault/src/main/java/com/solanamobile/seedvault/SigningResponse.java

```diff
@@ -49,7 +49,12 @@ protected SigningResponse(Parcel in) {
         for (int i = 0; i < size; i++) {
             mSignatures.add(in.createByteArray());
         }
-        mResolvedDerivationPaths = in.createTypedArrayList(Uri.CREATOR);
```

Comment:
`createTypedArrayList` can return null, but `mResolvedDerivationPaths` is marked `@NonNull`